File: JavaScript/Maths/partieDecimale.js
/**
 * @description Fonction qui permet d'isoler la partie décimale d'un nombre.
 * @param {number} n - Nombre à découper.
 * @returns {number} - Partie décimale du nombre.
 */
const partieDecimale = (n) => n.toString().split('.')[1];

// Test
console.log(partieDecimale(111.222)); // 222

File: Python/Maths/partie_decimale.py
def partie_decimale(n: float) -> int:
    """
    Description:
        Fonction qui permet d'isoler la partie décimale d'un nombre.
    
    Paramètres:
        n: {float} -- Le nombre dont on veut isoler la partie décimale.
    
    Retourne:
        {int} -- La partie décimale du nombre.

    Exemple:
        >>> partie_decimale(111.222)
        222
    """
    return str(n).split(".")[1]